AssetManagementTestProject/PageObj/LoginPage.cs

```diff
@@ -3,8 +3,8 @@
 namespace AssetManagementTestProject.PageObj;
 public class LoginPage : WebDriverAction
 {
-    private readonly string tfUsername = "//input[contains(@id, 'username')]";
-    private readonly string tfPassword = "//input[contains(@id, 'password')]";
+    public readonly string TfUsername = "//input[contains(@id, 'username')]";
+    public readonly string TfPassword = "//input[contains(@id, 'password')]";
     private readonly string btnLogin = "//button[contains(@type, 'submit')]";
```

Comment:
Change these properties as my comment in the ChangePasswordPage

Reply:
Done